README.md

```diff
@@ -135,8 +135,34 @@ See [rules_haskell_examples][] for examples of using these rules.
 
 [rules_haskell_examples]: https://github.com/tweag/rules_haskell_examples
 
+
 ## For `rules_haskell` developers
 
+### Reference a local checkout of `rules_haskell`
+
+When you develop on `rules_haskell`, you usually do it in the context
+of a different project that has `rules_haskell` as a `WORKSPACE`
+dependency, like so:
+
+```
+http_archive(
+    name = "io_tweag_rules_haskell",
+    strip_prefix = "rules_haskell-" + version,
+    sha256 = …,
+    urls = …,
+)
+```
+
```

Comment:
Why not using the bazel flag `override_repository`?

```
--override_repository \
   io_tweag_rules_haskell=/path/to/local/checkout
```

Reply:
Oh, that might be even better on occasions. I’ll add a reference to that as well.

Reply:
We should write a blog post about the most helpful bazel command line flags …

Reply:
Add the ref and I approve.

Reply:
Done.